CHANGELOG.md

```diff
@@ -3,13 +3,80 @@
+- collection/file-managers/gather-smart-ftp-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-softx-ftp-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-southriver-webdrive-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-staff-ftp-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-total-commander-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-turbo-ftp-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-ultrafxp-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-winscp-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-winzip-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-wise-ftp-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-ws-ftp-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- collection/file-managers/gather-xftp-information https://github.com/fireeye/capa-rules/pull/337 @re-fox
+- data-manipulation/compression/decompress-data-using-aplib https://github.com/fireeye/capa-rules/pull/313 @r3c0nst @mr-tz
+- host-interaction/bootloader/disable-code-signing https://github.com/fireeye/capa-rules/pull/367 @williballenthin
+- host-interaction/bootloader/manipulate-boot-configuration https://github.com/fireeye/capa-rules/pull/367 @williballenthin
+- host-interaction/driver/disable-driver-code-integrity https://github.com/fireeye/capa-rules/pull/367 @williballenthin
+- host-interaction/file-system/bypass-mark-of-the-web https://github.com/fireeye/capa-rules/pull/366 @williballenthin
+- host-interaction/network/domain/get-domain-information https://github.com/fireeye/capa-rules/pull/338 @recvfrom
+- host-interaction/session/get-logon-sessions https://github.com/fireeye/capa-rules/pull/338 @recvfrom
+- linking/runtime-linking/resolve-function-by-fin8-fasthash https://github.com/fireeye/capa-rules/pull/313 @r3c0nst @mr-tz
+- nursery/build-docker-image https://github.com/fireeye/capa-rules/pull/365 @williballenthin
+- nursery/create-container https://github.com/fireeye/capa-rules/pull/365 @williballenthin
+- nursery/encrypt-data-using-fakem-cipher https://github.com/fireeye/capa-rules/pull/314 @mike-hunhoff
+- nursery/list-containers https://github.com/fireeye/capa-rules/pull/365 @williballenthin
+- nursery/run-in-container https://github.com/fireeye/capa-rules/pull/365 @williballenthin
+- persistence/registry/appinitdlls/disable-appinit_dlls-code-signature-enforcement https://github.com/fireeye/capa-rules/pull/367 @williballenthin
+-
```

Comment:
This is a marker for the GitHub Action. It is an easy approach (As we can use `sed` to look for something like `-\n`. But if it is removed it will make the Action failing. Should we give it a try and look into another solution if this happens or do we want to use a different approach? Thoughts, ideas?

Reply:
I wonder how useful this actually is vs. the effort to create it.

I don't think anyone gets much benefit from this list. Can we use GitHub diff links (like https://github.com/fireeye/capa-rules/compare/X...Y) instead?

Reply:
I am not sure if I understand what you mean. We have this list in every release. It provides an easy-to-find list for users (the diff may include more things than just the new rules) and it allows us to mention/recognize rule contributors.

I find it painful to have to create this list manually. But the objective of this PR (together with https://github.com/fireeye/capa-rules/pull/374) is that we don't have to do this again and it is just updated with the sync GitHub Action. This also has the advantage that is makes it super easy to check the new rules between two dates (what I am doing in much inefficient ways at the moment).

This was also mentioned here: https://github.com/fireeye/capa/issues/457